scripts/cmake_setup.sh

```diff
@@ -48,5 +48,5 @@ make -j4
 make install
 set +e
 
-exit 0
+return 0
```

Comment:
I think this one should still be `exit` since it's a top level thing and not within a function? See https://stackoverflow.com/questions/4419952/difference-between-return-and-exit-in-bash-functions